### PR TITLE
[ONNX] Export gelu

### PR DESCRIPTION
**Pull request by neginraoof**
Added support for gelu in symbolic opset9 + op and ORT tests

### CODE REVIEW

**Review by BowenBao — COMMENTED** [review]
Looks good, some inline comments

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -1910,3 +1910,11 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
     return g.op("Multinomial", log_input,
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
+
+
+@parse_args('v')
+def gelu(g, self):
+    sqrt = g.op('Sqrt', g.op('Constant', value_t=torch.tensor(2, dtype=torch.float)))
+    erf = g.op('Erf', g.op('Div', self, sqrt))
```
[the remark this answers is not in the document]
Comment (neginraoof) [reply torch/onnx/symbolic_opset9.py]: What's the advantage here?

[the remark this answers is not in the document]
Comment (BowenBao) [reply torch/onnx/symbolic_opset9.py]: In general is to re-use the symbolic functions in case there are special handlings. For instance in the `Div` symbolic function there is code that does implicit casting for scalars. Calling directly `g.op(...)` misses that part.

File: torch/onnx/symbolic_opset9.py [hunk 2]
```diff
@@ -1911,3 +1911,10 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
     return g.op("Multinomial", log_input,
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
+
+
+def gelu(g, self):
+    _sqrt2 = 1.4142135623730951
+    erf = g.op('Erf', div(g, self, g.op('Constant', value_t=_sqrt2)))
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset9.py]:
```suggestion
    erf = g.op('Erf', div(g, self, g.op('Constant', value_t=torch.Tensor([_sqrt2]))))
```

Also the CI is failing

> Aug 21 06:40:09 test/onnx/test_operators.py::TestOperators::test_gelu FAILED             [  2%]

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 3]
```diff
@@ -759,6 +759,14 @@ def forward(self, x):
         x = torch.randn(4, 2, 3, requires_grad=True)
         self.run_test(NormModel(), x)
 
+    def test_gelu(self):
```
Comment (BowenBao) [inline test/onnx/test_pytorch_onnx_onnxruntime.py]: If all ops used for composition is supported, could we also add a test for Caffe2?

**Review by BowenBao — COMMENTED** [review]
Overall looks good, some minor suggestions, and please resolve CI failure.

File: test/onnx/test_pytorch_onnx_onnxruntime.py (hunk 3 above)
Comment (neginraoof) [reply test/onnx/test_pytorch_onnx_onnxruntime.py]: Sure, done

**Review by BowenBao — APPROVED** [review]
LG

**Review by dzhulgakov — APPROVED** [review]

**Comment by neginraoof** [comment]
@bddppq for review

**Comment by neginraoof** [comment]
cc @houseroad

**Comment by houseroad** [comment]
Fix the conflict?

**Comment by neginraoof** [comment]
cc @houseroad @bddppq for review

**Review by houseroad — APPROVED** [review]
Looks good.
